docker/auth/realm.json

```diff
@@ -46,6 +46,9 @@
   "users": [
     {
       "username": "people",
+      "email": "people@people.world",
+      "firstName": "John",
+      "lastName": "Doe",
```

Comment:
Should we add similar mocked values on the other users used by e2e tests?

Reply:
ok. Can you point to where are those users?

Reply:
Okay, then let's ignore my comment !